### PR TITLE
BIOIN-1909-ugbio-cnv-docker-missing-tabix

### PR DESCRIPTION
**Pull request by TammushB**
replace tabix with bcftools

### CODE REVIEW

**Comment by TammushB** [comment]
@avigailmo  instead of installing tabix , I choose to use bcftools (already installed) to index the vcf.

File: src/cnv/ugbio_cnv/convert_cnv_results_to_vcf.py [hunk 1]
```diff
@@ -153,8 +153,12 @@ def run(argv):  # noqa: C901, PLR0912, PLR0915 #TODO: Refactor this function
 
         vcf_out.close()
 
-        cmd = f"tabix -p vcf {outfile}"
-        os.system(cmd)  # noqa: S605
+        try:
+            cmd = ["bcftools", "index", "-t", outfile]
+            subprocess.check_call(cmd)
+        except subprocess.CalledProcessError as e:
```
Comment (avigailmo) [inline src/cnv/ugbio_cnv/convert_cnv_results_to_vcf.py:159]: 👍

File: src/cnv/tests/system/test_convert_cnv_results_to_vcf.py [hunk 2]
```diff
@@ -52,5 +52,5 @@ def test_convert_cnv_results_to_vcf(self, tmpdir, resources_dir):
             ]
         )
 
-        out_vcf_file = pjoin(tmpdir, sample_name + ".cnv.vcf.gz")
+        out_vcf_file = pjoin(tmp_path, sample_name + ".cnv.vcf.gz")
```
Comment (avigailmo) [inline src/cnv/tests/system/test_convert_cnv_results_to_vcf.py:55]: can you add verification of the vcf index creation?

**Review by avigailmo — CHANGES_REQUESTED** [review]
please add check that vcf index created in the test

**Pull request merged**